src/Components/ProductTile.js

```diff
@@ -1,6 +1,15 @@
+import React from 'react';
+import { ReactSession } from 'react-client-session';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8496218)
```suggestion
import { ReactSession } from "react-client-session";
```